api/v1beta2/ibmvpcmachine_types.go

```diff
@@ -36,10 +39,23 @@ type IBMVPCMachineSpec struct {
 	// Name of the instance.
 	Name string `json:"name,omitempty"`
 
+	// CatalogOffering is the Catalog Offering OS image which would be installed on the instance.
+	// An OfferingCRN or VersionCRN is required, the PlanCRN is optional.
+	// +optional
+	CatalogOffering *IBMCloudCatalogOffering `json:"catalogOffering,omitempty"`
+
+	// DedicatedHost is the name of the underlying provisioned host in the VPC on which the instance will be created.
+	// +optional
+	DedicatedHost *VPCResource `json:"dedicatedHost,omitempty"`
+
 	// Image is the OS image which would be install on the instance.
 	// ID will take higher precedence over Name if both specified.
 	Image *IBMVPCResourceReference `json:"image"`
 
+	// LoadBalancerPoolMembers is the set of IBM Cloud VPC Load Balancer Backend Pools the machine should be added to as a member.
+	// +optional
+	LoadBalancerPoolMembers []VPCLoadBalancerBackendPoolMember `json:"loadBalancerPoolMembers,omitempty"`
```

Comment:
We used to attach the machine to all the loadbalancers defined in cluster.Loadbalancer, Do you plan to avoid that and register to only loadbalancers set here?

What  will happen when this is omitted?

Reply:
The current code has some hardcoded values and expectations on attaching Machines to LB's.
I am trying to avoid doing that, especially since it is currently hardcoded to one LB.
I would expect that all machines CAPI creates then would be added into LB pools then, if we rely on Cluster definitions, which I do not expect is valid. Is that a fair assumption, if we rely on the Cluster resource?

Reply:
We can do this way as well, but my point is eventually the lb referred by VPCLoadBalancerBackendPoolMember.Loadbalancer is created by cluster reconciler, So why can't we directly use it.